songChulJin/app.js

```diff
@@ -0,0 +1,52 @@
+// Built-in package
+const http = require("http");
+
+// 3rd-party package
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+
```

Comment:
- dotenv 는 최상단에 위치하여야 합니다 ! 
- 더불어 개발 시 사용한 주석은 지워주세요 !

```suggestion
require("dotenv").config();

const http = require("http");
const express = require("express");
const cors = require("cors");
const morgan = require("morgan");
```